UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/DistributionService.cs

```diff
@@ -105,6 +105,19 @@ public bool SaveCommandDistributorRequest(string subject, HttpStatusCode? httpSt
         {
             return s_commandDistributionList.LastOrDefault(a => a.Subject == subject);
         }
+
+        public void ClearDistributorQueue()
+        {
+            var distQueue = s_recordDistributorRequestQueue.ToList();
+
+            if (distQueue.Count > 0)
+            {
+                foreach (var dist in distQueue)
+                {
+                    s_recordDistributorRequestQueue.Remove(dist);
```

Comment:
Can you add a log entry here, so that we know that the queue has been manually cleared?

Reply:
Commit 6ff162b6dd49ea423bacc77c0bd87df273864d88 address this

Reply:
Great, thanks!